hbase-client/src/main/java/org/apache/omid/transaction/HBaseTransactionManager.java

```diff
@@ -292,7 +268,7 @@ public ConflictDetectionLevel getConflictDetectionLevel() {
             get.addColumn(family, shadowCellQualifier);
             get.setMaxVersions(1);
             get.setTimeStamp(startTimestamp);
-            Result result = hBaseCellId.getTable().get(get);
+            Result result = (tableAccessWrapper != null) ? tableAccessWrapper.get(get) : hBaseCellId.getTable().get(get);
```

Comment:
why should this be null? Maybe its better to pass "hBaseCellId.getTable().get(get);" to the constructor code